ground/gcs/src/plugins/sysalarmsmessaging/sysalarmsmessagingplugin.cpp

```diff
@@ -89,42 +90,38 @@ bool SysAlarmsMessagingPlugin::initialize(const QStringList &arguments, QString
  */
 void SysAlarmsMessagingPlugin::updateAlarms(UAVObject* systemAlarm)
 {
-    Q_ASSERT(systemAlarm->getObjID() == SystemAlarms::OBJID);
```

Comment:
Ps i wont consider this a blocker since run time errors will only happen in non clean builds. Clean builds are fine.